packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useNextItem.tsx

```diff
@@ -1,11 +1,10 @@
-import { Item } from '@openmsupply-client/common';
-import { toItem } from '@openmsupply-client/system';
+import { ItemRowFragment, toItemRow } from '@openmsupply-client/system';
 import { useOutboundRows } from '../../../api';
 
 export const useNextItem = (
   currentItemId?: string
-): { next: Item | null; disabled: boolean } => {
-  const next: Item | null = null;
+): { next: ItemRowFragment | null; disabled: boolean } => {
+  const next: ItemRowFragment | null = null;
   const disabled = true;
 
   const { items } = useOutboundRows();
```

Comment:
this now returns a single item in the array and so the next button is always disabled. 
My suggestion - change line 208 in `OutboundShipment/api/hooks.ts` now that `itemId` isn't a thing

```
    return Object.entries(groupBy(lines.nodes, 'item.id')).map(
```